we_count/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
We typically use 4 spaces for our projects. Is there a specific reason to use 2 spaces for this work?

Reply:
No reason just default. Will change for next pr